### PR TITLE
Added Confirmation Before Delete Task

### PR DESCRIPTION
**Pull request by 1hanzla100**

### CODE REVIEW

**Comment by 1hanzla100** [comment]
@shacker  Review Pull Request and Merge it with master branch

File: todo/templates/todo/task_detail.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h3 class="card-title">{{ task.title }}</h3>
           <form method="post" action="{% url "todo:delete_task" task.id %}" role="form" class="d-inline">
             {% csrf_token %}
             <div style="display:inline;">
-              <button class="btn btn-danger btn-sm" type="submit" name="submit_delete">
+              <button class="btn btn-danger btn-sm" type="submit" onclick="return confirm('Are you sure to delete task?');" name="submit_delete">
```
Comment (shacker) [inline todo/templates/todo/task_detail.html:56]: Please make it say "Are you sure you want to delete this task?"

**Review by shacker — CHANGES_REQUESTED** [review]
Hi - Can you please make the language change requested, and also post a couple of screenshots in the PR description, showing what happens when you click either of the two buttons? (as proof that it does delete what it should delete, and doesn't delete if you cancel out). Thanks.

**Comment by shacker** [comment]
@1hanzla100 We never heard back from after the code review above. Did you still want to have this merged? Please respond.

**Comment by shacker** [comment]
@1hanzla100 Nevermind, I'll just bring it in and change it myself. Thanks!

**Pull request merged**